flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/controller/FlinkSessionJobController.java

```diff
@@ -162,8 +162,19 @@ public DeleteControl cleanup(FlinkSessionJob sessionJob, Context context) {
 
     private InformerEventSource<FlinkDeployment, FlinkSessionJob> createFlinkDepInformerEventSource(
             String name) {
+        var sharedIndexInformer =
+                ALL_NAMESPACE.equals(name)
+                        ? kubernetesClient
+                                .resources(FlinkDeployment.class)
+                                .inAnyNamespace()
+                                .runnableInformer(0)
+                        : kubernetesClient
+                                .resources(FlinkDeployment.class)
+                                .inNamespace(name)
```

Comment:
I think we could pass the informer as the first parameter of `createFlinkDepInformerEventSource` like the `OperatorUtils#createJmDepInformerEventSource`. By this, we do not have to assume the name is the namespace

Reply:
updated